includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -67,14 +67,19 @@ public function output_analytics_tracking_script() {
 		echo $analytics_script;
 	}
 
-	public function load_ecommerce_analytics( $order_id ) {
+	public function load_ecommerce_analytics($order_id ) {
```

Comment:
[minor] According to [wordpress php styling guide](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage) we should leave a whitespace before and after the function arguments. So I suggest to revert this change.